scripts/test_publish.sh

```diff
@@ -34,7 +34,7 @@ trap 'cd "$LINERA_DIR"; git checkout -f HEAD .cargo/config.toml' EXIT
 # Initialize the git repository for the index if needed. Ideally, we'd like to use `cargo
 # index init` first but the tool refuses to update an existing directory.
 git init "$REGISTRY"/index || true
-(cd "$REGISTRY"/index; git add .; git commit -m 'update registry')
+(cd "$REGISTRY"/index; git add .; git commit -m 'update registry' ||:)
```

Comment:
Is that the same as `|| true`? Do we need this?

Reply:
We need it in order to re-use a registry that may already exist.  Otherwise this command will fail; the outcome of that is that the script as a whole succeeds, but actually doesn't test anything (!).

Reply:
Would `--allow-empty` be more precise?

Reply:
Works for me!